app/utils/dndManager.js

```diff
@@ -5,7 +5,7 @@ class DndManager extends EventEmitter {
     super()
     this.settings = settings
     this.monitorDnd = settings.get('monitorDnd')
-    this.timer = null
+    this.dndTimer = null
```

Comment:
One more thing: let's rename this back to `timer` :) not really important but... you know... :) 

could you also squash all your commits to one? if not, I can also do that when merging :)